### PR TITLE
SE-837-DemoSetup-script

### PR DESCRIPTION
**Pull request by ShaneDuncan602**

### CODE REVIEW

File: docs/userGuides/README.md [hunk 1]
```diff
@@ -46,61 +46,25 @@ And you are done!
...
-Make sure that all the srcipts ran successfully, and then:
+### Local Deployments(Protocol and/or Examples)
 
-```bash 
-forge test --ffi --rpc-url http://127.0.0.1:8545 
-```
+For local deployments, there are two ways to deploy the protocol and application examples:
 
+1. [Start an instance of anvil and run the individual scripts](./deployment/DEPLOY-LOCAL-SCRIPTS.md)
+2. [Start an instance of anvil and run an all-in-one script deploys the protocol and deploys example assets](./deployment/DEPLOY-LOCAL-SCRIPTS.md).
```
Comment (TJ-Everett) [inline docs/userGuides/README.md]: all-in-one script * that * deploys the protocol

File: docs/userGuides/README.md [hunk 2]
```diff
@@ -119,13 +83,23 @@ These scripts deploy the contracts that are specific for applications, emulating
...
 
+---
+
 ### Testnet Deployment
 
-coming soon...
+For Testnet deployments, there are two ways to deploy the protocol and application examples:
+
+1. [Start an instance of anvil and run the individual scripts](./deployment/DEPLOY-TESTNET-SCRIPTS.md)
+2. [Start an instance of anvil and run an all-in-one script deploys the protocol and deploys example assets](./deployment/DEPLOY-TESTNET-ALL.md).
```
Comment (TJ-Everett) [inline docs/userGuides/README.md]: all-in-one script * that * deploys the protocol

File: docs/userGuides/README.md [hunk 3]
```diff
@@ -119,13 +83,23 @@ These scripts deploy the contracts that are specific for applications, emulating
...
+
+---
 
 ### Mainnet Deployment
 
-coming soon...
+For Mainnet deployments, there are two ways to deploy the protocol and application examples:
+
+1. [Start an instance of anvil and run the individual scripts](./deployment/DEPLOY-MAINNET-SCRIPTS.md)
+2. [Start an instance of anvil and run an all-in-one script deploys the protocol and deploys example assets](./deployment/DEPLOY-MAINNET-ALL.md).
```
Comment (TJ-Everett) [inline docs/userGuides/README.md]: all-in-one script * that * deploys the protocol

**Review by TJ-Everett — APPROVED** [review]
This is great work Shane! Approved for me.

**Review by Palmerg4 — APPROVED** [review]
This is great, very easy to follow. Approved for me

**Review by TJ-Everett — APPROVED** [review]
Loving the simplified changes. Approved

**Pull request merged**